addons/mail/models/mail_thread.py

```diff
@@ -3114,8 +3111,90 @@ def send_notifications():
 
         return True
 
-    def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, model_description=False,
-                                                   force_email_company=False, force_email_lang=False):
+    def _notify_get_recipients_and_context_iterator(
```

Comment:
_notify_get_classified_recipients_iterator?

"and_context" is a bit strange, I feel like adding "classified" respects other naming more.

Reply:
Ok.